sys/include/net/gcoap.h

```diff
@@ -502,13 +502,16 @@ void gcoap_register_listener(gcoap_listener_t *listener);
  * @param[out] buf      Buffer containing the PDU
  * @param[in] len       Length of the buffer
  * @param[in] code      Request code: GCOAP_[GET|POST|PUT|DELETE]
- * @param[in] path      Resource path, *must* start with '/'
```

Comment:
Why did you remove this now. It is always good to have redundant documentation for people with tunnel vision ;-).